### PR TITLE
Updating time section 

### PR DESCRIPTION
**Pull request by hayesla**
In this PR I tried to address Issue #99 

I'm not too sure if its exhaustive but tried to hit on the points of what astropy.time offers and why it is used. I've also updated the reference to 'high precision time format' and 'modern astronomical time format' to just 'specific scientific time class'.

### CODE REVIEW

File: sections/8-units.tex [hunk 1]
```diff
@@ -20,8 +20,9 @@ \subsection{Units and Time Scales}
...
-Thus, functions that require precise times (e.g., for ephemeris calculations) can ensure that there is no confusion about the input.
+Solar data analysis requires certain functionality to manipulate and represent times and dates, such as having support for leap seconds and the ability to represent specific time scales and formats. 
+This requirement goes beyond that provided by the standard \python \package{datetime} subpackage.
+For this reason, SEP-0008 \citep{sep-0008} was mandated to adopt the use of the \package{astropy.time} subpackage to represent a date or time throughout the \sunpypkg code base.
+The \package{astropy.time} subpackage provides the necessary support non-UTC time scales used in some solar data sets such as International Atomic Time (TAI), and also provides functionality for representing time formats used in solar physics such as Julian day and Unix time. 
+It also provides support for creating custom time scales and formats. 
+Furthermore the adoption of the \package{astropy.time} within \sunpypkg provides the necessary support for leap seconds, light travel calculations, and high precision time representation.
```
Comment (ehsteve) [inline sections/8-units.tex]: Do you want to add something about being able to accurately compare the timing of solar events to astrophysical events? I'm not sure what the use case would be. Maybe being able to accurately track solar events through the solar system compared to planetary bodies or accurately compare inputs to the Earth's magnetosphere by solar and extra solar events? Another one might be to be able to differentiate events in x-ray data that are solar compared to those that are extra-solar like pulsar bursts.

**Review by ehsteve — CHANGES_REQUESTED** [review]
One suggestion to add example for solar and astro to use the same high-precision time format so that events can be compared.

**Comment by hayesla** [comment]
i've now updated the text to make reference to this, however I have not actually included a use-case example - I'm not too sure of one! How do you think it reads now?

**Comment by hayesla** [comment]
ping @ehsteve @ayshih

**Review by ehsteve — APPROVED** [review]

**Pull request merged**